charts/docker-hub-rate-limit-exporter-chart/templates/deployment.yaml

```diff
@@ -1,55 +1,59 @@
+{{- if .Values.deployment.enabled }}
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "docker-hub-rate-limit-exporter.fullname" . }}
   labels:
     {{- include "docker-hub-rate-limit-exporter.labels" . | nindent 4 }}
 spec:
-  replicas: {{ .Values.replicaCount }}
+  replicas: {{ .Values.deployment.replicaCount }}
   selector:
     matchLabels:
       {{- include "docker-hub-rate-limit-exporter.selectorLabels" . | nindent 6 }}
-      {{- if .Values.podLabels }}
-      {{- toYaml .Values.podLabels | nindent 6 }}
-      {{- end }}
   template:
     metadata:
+      labels:
+        {{- include "docker-hub-rate-limit-exporter.selectorLabels" . | nindent 8 }}
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
         checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
-        {{- with .Values.podAnnotations }}
-        {{- toYaml . | nindent 8 }}
+        {{- if .Values.podAnnotations }}
+        {{- toYaml .Values.podAnnotations | nindent 8 }}
         {{- end }}
-      labels:
-        {{- include "docker-hub-rate-limit-exporter.selectorLabels" . | nindent 8 }}
     spec:
-      {{- with .Values.imagePullSecrets }}
+      {{- if .Values.imagePullSecrets }}
       imagePullSecrets:
-        {{- toYaml . | nindent 8 }}
+{{ toYaml .Values.imagePullSecrets | indent 8 }}
       {{- end }}
       serviceAccountName: {{ include "docker-hub-rate-limit-exporter.serviceAccountName" . }}
+      {{- if .Values.securityContext }}
       securityContext:
-        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+{{ toYaml .Values.securityContext | indent 8 }}
```

Comment:
You change the variable from .Values.podSecurityContext to .Values.securityContext, which seems to be an unnecessary breaking change.

Moreover, the default naming (e.g. when running helm create) is using `.Values.podSecurityContext` and `.Values.securityContext` instead of your suggested change. 

Was there a specific reason to change the naming? If not, do you mind reverting it?